routing/hostname_pattern.rst

```diff
@@ -193,7 +193,8 @@ instance, if you want to match both ``m.example.com`` and
             <?xml version="1.0" encoding="UTF-8" ?>
             <routes xmlns="http://symfony.com/schema/routing"
                 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-                xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
+                xsi:schemaLocation="http://symfony.com/schema/routing
```

Comment:
I searched on the documentation (all of it), the https is never used in XML examples. Is it normal to do it just in this example ?